pkg/apis/experimental/types.go

```diff
@@ -639,3 +639,36 @@ type ClusterAutoscalerList struct {
 
 	Items []ClusterAutoscaler `json:"items"`
 }
+
+// Selector is a label query over a set of API objects. MatchLabels and
+// MatchExpressions are ANDed. An empty Selector matches all objects. A
+// null Selector matches no objects.
+type Selector struct {
+	// MatchLabels is a list of labels.
+	MatchLabels map[string]string `json:"matchLabels,omitempty"`
+	// MatchExpressions is a list of requirement selectors.
+	MatchExpressions []SelectorRequirement `json:"matchExpressions,omitempty"`
+}
+
+// SelectorRequirement is a selector that contains values, a key
+// and an operator that relates the key and values.
+type SelectorRequirement struct {
+	// Key is the label key that the selector applies to.
+	Key string `json:"key"`
+	// Operator represents a key's relationship
+	// to a set of values.
+	Operator SelectorOperator `json:"operator"`
+	// StringValues is a set of string values. If the operator is In or NotIn,
+	// the values set must be non-empty.
+	StringValues []string `json:"stringValues,omitempty"`
```

Comment:
Actually, let's just go with Values/values. We don't plan to add ints in the foreseeable future.